sagemaker-core/src/sagemaker/core/helper/session_helper.py

```diff
@@ -1865,6 +1865,30 @@ def expand_role(self, role):
         if "/" in role:
             return role
         return self.boto_session.resource("iam").Role(role).arn
+    
+    
+    def logs_for_job(self, job_name, wait=False, poll=10, log_type="All", timeout=None):
```

Comment:
Wanted to check on what this utility is used for?

Reply:
It seem like we are replacing:
v2 logic: `job.get_estimator().logs()`

v3 logic: `model_trainer.sagemaker_session.logs_for_job(model_trainer._latest_training_job.training_job_name, wait=True)`

The V3 experience looks pretty bad to me and since this is not an existing v2 parity issue - Can we think about tie-ing the get logs functionality to training queue job or model trainer directly? We can also think through on this and not make a decision on this for 1st release. Lets use utility method within example notebooks (replicate _logs_for_job funtionality within notebook as a standalone utility)? This would not break customers using the notebook and we can make a right offering for logs exposure and update the notebook.